bin/smoke-test.sh

```diff
@@ -655,6 +687,9 @@ while getopts "h?t:v:p:r:u:s:fxlLCcd2" opt; do
         v)
             vo=$OPTARG
             ;;
+        o)
+            OIDC_AGENT_ACCOUNT=$OPTARG
+            ;;
         p)
             THIRDPARTY_UNAUTHENTICATED_URL=$OPTARG
             ;;
```

Comment:
[github doesn't let me make this comment further down the file]

Line 748 has:

    [ "$tokenType" = "scitoken" ] && [ "$sciTokenServer" = "" ] && fatal "No SciToken server"

There should be a similar test to verify that if `tokenType` is `WlcgToken` then `OIDC_AGENT_ACCOUNT` has also been specified.

Reply:
Ah - indeed, and the above is actually a bug too.  Should be `[ "$tokenType" = "SciToken" ]`; I'll fix it.